package.json

```diff
@@ -20,6 +20,7 @@
     "test:cli": "node tests/runner",
     "test:inspect": "node --inspect --debug-brk tests/runner",
     "test:packages": "node scripts/run-packages-spec.js",
+    "test:universal": "UNIVERSAL=true node tests/e2e_runner.js",
```

Comment:
These tests are not being ran in the CI atm, you might have to fiddle with `travis.yml` to see them run.